package.json

```diff
@@ -23,6 +23,7 @@
     "babel-loader": "^9.1.2",
     "css-loader": "^6.7.3",
     "html-webpack-plugin": "^5.5.0",
+    "jest": "^29.5.0",
```

Comment:
Nice to have `jest` for the testing 🧪 but you must also add the `test` entry in your `scripts` object. Also, update the `README.md` file to explain how to run the tests properly. 👀